piptools/utils.py

```diff
@@ -71,13 +71,22 @@ def make_install_requirement(name, version, extras, constraint=False):
         constraint=constraint)
 
 
+def is_url_requirement(ireq):
+    """
+    Finds if a requirement is a URL
+    """
+    return bool(ireq.link and 'pypi.python.org' not in ireq.link.url and not ireq.link.url.startswith('file'))
```

Comment:
Is there a reason to exclude `pypi.python.org` here? 
If its needed to exclude pypi-hosted packages, the we need to exclude the configured pypi url too for users using their local devpi instances and such.